packages/ui/src/constants.ts

```diff
@@ -189,6 +189,19 @@ export const networkList = {
   }
 }
 
+export const polkadotNetworksAndParachains = [
+  'polkadot',
+  'hydradx',
+  'astar',
+  'moonbeam',
+  'assethub-dot',
+  'bifrost-dot',
+  'interlay',
+  'phala'
+]
+export const kusamaNetworksAndParachains = ['kusama', 'khala', 'assethub-ksm', 'moonriver']
+export const testChains = ['rococo', 'acala', 'rhala testnet', 'local']
```

Comment:
acala is a Polkadot chain :)
For the order, I'd suggest to have the relay-chain and asset-hub first (system parachain) the rest in alphabetical order.

Reply:
Yeah, I need help here to validate if everything is correct

Reply:
@Tbaut is it a correct order?
```js
export const polkadotNetworksAndParachains = [
  'polkadot',
  'acala',
  'moonbeam',
  'hydradx',
  'phala',
  'bifrost-dot',
  'assethub-dot',
  'astar',
  'interlay'
]
export const kusamaNetworksAndParachains = ['kusama', 'khala', 'moonriver', 'assethub-ksm']
export const testChains = ['rococo', 'rhala testnet', 'local']
```

Reply:
BTW why we don't have a westend? I thought it was a popular test net.  😃

Reply:
There was no subsquid archive available for Westend (those are the archives supplied by subsquid) and although they said it's planned, it didn't happen yet. So it means that we don't have any indexer ready for it.

Alphabetical order starting after the relay + system chains should make it so
```ts
export const polkadotNetworksAndParachains = [
  'polkadot',
  'assethub-dot',
  'acala',
  'astar',
  'bifrost-dot',
  'hydradx',
  'interlay'
  'moonbeam',
  'phala',
]
export const kusamaNetworksAndParachains = ['kusama', 'assethub-ksm', 'khala', 'moonriver']
export const testChains = ['rococo', 'rhala testnet', 'local']